docs/source/dev-and-test/setup-run-node.md

```diff
@@ -43,36 +43,58 @@ Ian Worrall of [Encrypted Labs](http://www.encryptedlabs.com/) wrote a document
 ## Option C: Using a Local Dev Machine and Docker
 
 You need to have recent versions of [docker engine](https://docs.docker.com/engine/installation/#installation)
-and [docker-compose](https://docs.docker.com/compose/install/), and possibly
-[docker-machine](https://docs.docker.com/machine/install-machine/),`if you are
-running on Mac OS X or Windows, for instance.
+and [docker-compose](https://docs.docker.com/compose/install/).
 
 Build the images:
 
 ```bash
-$ docker-compose build
+docker-compose build
 ```
 
 Start RethinkDB:
 
 ```bash
-$ docker-compose up -d rdb
+docker-compose up -d rdb
 ```
 
+The RethinkDB web interface should be accessible at <http://localhost:58080/>.
+Depending on which platform, and/or how you are running docker, you may need
+to change `localhost` for the `ip` of the machine that is running docker. As a
+dummy example, if the `ip` of that machine was `0.0.0.0`, you would accees the
+web interface at: <http://0.0.0.0:58080/>.
+
 Start a BigchainDB node:
 
 ```bash
-$ docker-compose up -d bdb
+docker-compose up -d bdb
 ```
 
 You can monitor the logs:
 
 ```bash
-$ docker-compose logs -f bdb
+docker-compose logs -f bdb
 ```
 
 If you wish to run the tests:
 
 ```bash
-$ docker-compose run --rm bdb py.test -v -n auto
+docker-compose run --rm bdb py.test -v -n auto
+```
+
+A quick check to make sure that the BIgchainDB server API is operational:
+
+```bash
+curl $(docker-compose port bdb 9984)
```

Comment:
It's not very clear to me what this does. Can you say a bit more about it?

Could we send a `curl` GET to `http://bdb:9984/api/v1` instead?